app/src/main/java/com/github/dedis/student20_pop/PoPApplication.java

```diff
@@ -65,43 +61,17 @@ public void onCreate() {
 
     appContext = getApplicationContext();
 
-    SharedPreferences sp = this.getSharedPreferences(TAG, Context.MODE_PRIVATE);
-
-    // Verify if the information is not present
-    if (person == null) {
-      // Verify if the user already exists
-      if (sp.contains(SP_PERSON_ID_KEY)) {
-        // Recover user's information
-        String id = sp.getString(SP_PERSON_ID_KEY, "");
-        String authentication = PrivateInfoStorage.readData(this, id);
-        if (authentication == null) {
-          person = new Person(USERNAME);
-          Log.d(TAG, "Private key of user cannot be accessed, new key pair is created");
-          if (PrivateInfoStorage.storeData(this, person.getId(), person.getAuthentication()))
-            Log.d(TAG, "Stored private key of organizer");
-        } else {
-          person = new Person(USERNAME, id, authentication, new ArrayList<>());
-        }
-      } else {
-        // Create new user
-        person = new Person(USERNAME);
-        // Store private key of user
-        if (PrivateInfoStorage.storeData(this, person.getId(), person.getAuthentication()))
-          Log.d(TAG, "Stored private key of organizer");
-      }
-    }
-
-    activateTestingValues(); // comment this line when testing with a back-end
+    // activateTestingValues(); // comment this line when testing with a back-end
   }
 
-  @SuppressLint("ApplySharedPref")
-  @Override
-  public void onTerminate() {
-    super.onTerminate();
-    SharedPreferences sp = this.getSharedPreferences(TAG, Context.MODE_PRIVATE);
-    // Use commit instead of apply for information to be stored immediately
-    sp.edit().putString(SP_PERSON_ID_KEY, person.getId()).commit();
-  }
+//  @SuppressLint("ApplySharedPref")
+//  @Override
+//  public void onTerminate() {
+//    super.onTerminate();
+//    SharedPreferences sp = this.getSharedPreferences(TAG, Context.MODE_PRIVATE);
+//    // Use commit instead of apply for information to be stored immediately
+//    sp.edit().putString(SP_PERSON_ID_KEY, person.getId()).commit();
+//  }
```

Comment:
if this and the above is dead code, do you want to remove it?

Reply:
Going to remove all dead code in a later commit